### DISCUSSION TITLE
Есть возможность убрать из сообщения техническую часть?

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#489] [Как это сделать?] [answered]
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/114401808/845ce40c-735d-4483-9b5b-5ce9945162e3)

Применяется функция errorException.

**Comment by erpic-ru** [comment]
Эта часть видна только пользователю с ролью Создатель

**Reply by vldekov** [reply, marked as answer]
Точно, спасибо!

**Reply by ttm-alexeypolunin** [reply]
Нам надо добавить в user режим, который бы как-раз включал ее для пользователя. Или еще лучше отслеживал такие ошибки и отправлял бы их в специальный отстойник для дальнейшего анализа.
